### ISSUE TITLE
feedback for rules description in image

### ISSUE THREAD

**Issue by wolftune** [nardo/Equal.Vote#5]
I think the "yes, it's really that simple" is not good. If you have to say it, it gives people the wrong impression. It's like saying "it's not a scam!" which makes people start thinking about scams, the opposite of the intention.

This is objectively far more complex than the current system, so asserting simplicity as though that's the feature is misguided.

For the wording, I think "automatic runoff" is actually a better term than "instant runoff" not only to differentiate from IRV, but because the automatic nature of it is far more important than the instant aspect.

How about:

"The two candidates scored highest by voters overall go to an automatic runoff. In the runoff, if you scored one candidate higher than the other, then they get your vote."

?

I'd consider then saying "So with score runoff, we select from the candidates with the most overall approval, and we maximize every voter's chance to express their preference among the top two without losing the chance to express their judgment of other candidates." or something similar. Point is to emphasize how great the process is.

We could say also something like, "the results are clear and transparent, and, unlike strictly ranked voting, everyone can easily understand how any one ballot or any precinct or regional results relate to the overall result."

**Comment by nardo** [comment]
Disagree. "Yes, it's really this simple" refers to the overall solution for our horribly complicated and misrepresentative two-step voting process. And since the first FUD people always sling is "OMG scoring is SO COMPLICATED", claiming the simple ground up front is important.

Instant runoff is used precisely because it is a known term.

The two sentence version you supply, "The two candidates scored highest by voters overall go to an automatic runoff. In the runoff, if you scored one candidate higher than the other, then they get your vote." uses more words to still not answer the question of "what happens if I score them both the same."

**Comment by wolftune** [comment]
I understand the stupid FUD, but readers who have never heard of the FUD, the "yes, it's this simple" risks bringing FUD to mind when they weren't aware of it. I *support* keeping some emphasis on how nice and simple this is, but it would be nice to do it in a place and fashion that couldn't be seen as defensive.

"instant runoff" is a known term but generally means the exact IRV system. I also think "automatic runoff" is clearer to everyone who isn't familiar with the terms. So I think "instant runoff" confuses knowledgeable people *and* naive people.

Regarding wording, I think we could do better than my suggestion, but more words in two sentences is superior to a long sentence. It's not about word-count, it's about the mental challenge of keeping more concepts in working memory before closing off each chunk.

**Comment by nardo** [comment]
Agreed re: automatic runoff. Site and text now reflect that change.
